fluent-package/make-index-html.rb

```diff
@@ -0,0 +1,45 @@
+require 'erb'
+require 'find'
+require 'pathname'
+require 'optparse'
+
+options = {
+  :site => "https://fluentd.cdn.cncf.io",
+  :verbose => false
+}
+
+opt = OptionParser.new
+opt.on("-s", "--site [URL]") { |v| options[:site] = v }
+opt.on("-v", "--verbose") { |v|
+  options[:verbose] = v
+}
```

Comment:
```suggestion
opt.on("-v", "--verbose") { options[:verbose] = true }
```

It can be simpler like this.